.github/workflows/dev.yml

```
@@ -0,0 +1,44 @@
+name: DEV
+
+on:
+  push:
+  pull_request:
+jobs:
+  build-deploy:
```

I think it's awesome that this is running on `push` and `pull_request` but it's not really deploying so makes sense to tweak the name IMO
```suggestion
  push:
  pull_request:
jobs:
  build-test:
```